src/main/java/com/autotune/common/performanceProfiles/perfProfileInterface/PerfProfileInterface.java

```diff
@@ -0,0 +1,8 @@
+package com.autotune.common.performanceProfiles.perfProfileInterface;
+
+public interface PerfProfileInterface {
```

Comment:
Please add some javadoc's here to explain what it is doing

Reply:
Added